files/image_config/caclmgrd/caclmgrd

```diff
@@ -368,14 +376,18 @@ class ControlPlaneAclManager(object):
                     for ip_protocol in ip_protocols:
                         for dst_port in dst_ports:
                             rule_cmd = "ip6tables" if table_ip_version == 6 else "iptables"
-                            rule_cmd += " -A INPUT -p {}".format(ip_protocol)
+                            if ip_protocol != "any":
+                                rule_cmd += " -A INPUT -p {}".format(ip_protocol)
+                            else:
+                                rule_cmd += " -A INPUT "
 
                             if "SRC_IPV6" in rule_props and rule_props["SRC_IPV6"]:
                                 rule_cmd += " -s {}".format(rule_props["SRC_IPV6"])
                             elif "SRC_IP" in rule_props and rule_props["SRC_IP"]:
                                 rule_cmd += " -s {}".format(rule_props["SRC_IP"])
 
-                            rule_cmd += " --dport {}".format(dst_port)
+                            if dst_port != "0":
```

Comment:
I think this could use a comment above, specifying that port 0 is a reserved/unused port, so we use it to signify that we want to apply the rule to all ports.

Reply:
done.